hugo/content/_index.html

```diff
@@ -178,7 +118,7 @@ <h1>100% Kubernetes</h1>
 					</div>
 					<div class="description">
 						One major goal of <a href="https://github.com/gardener/gardener" target="_blank">Gardener</a> is to provide
-						Kubernetes clusters which completely satisfy the requirements of the <a target="_blank" href="https://www.cncf.io/">CNCF</a>.
+						Kubernetes clusters which completely satisfy the requirements of the <a target="_blank" href="/https://www.cncf.io/">CNCF</a>.
```

Comment:
Remove leading slash to fix broken href

href="/https://www.cncf.io/"

Some of the links on this page open a new browser tab some don't. This seems inconsistent.

Reply:
3242f9f6f41f5b6207014bbd64684eba6a4d2395